cpp/include/cudf/io/functions.hpp

```diff
@@ -319,6 +319,43 @@ std::unique_ptr<table> read_parquet(
     read_parquet_args const& args,
     rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
+/**
+ * @brief Settings to use for `write_parquet()`
+ */
+struct write_parquet_args {
+  sink_info sink;
+
+  /// Specify the compression format to use
+  compression_type compression = compression_type::AUTO;
+  /// Specify the level of statistics in the output file
+  int stats_level = 1;
```

Comment:
What are the possible levels?  Is this the same as the `enum statistics_freq`?

Reply:
Yes (see further below), 0=none, 1=rowgroup, 2=page. I would expect that we'd only use 0 or 1 in practice (in fact just the default of 1, as there is not much benefit to turning off statistics).

Reply:
Changed it back to use statistics_freq to make this more clear